source/includes/candidate-ingestion/_candidates.md

```diff
@@ -170,17 +173,18 @@ referrer.email | String | Mixed | Used to match this referrer with an existing G
 referrer.first_name | String | Mixed | Used to create a new ‘Referrer’ in Greenhouse if referrer.email does not match an existing user. Only required if referrer is included.
 referrer.last_name  | String | Mixed | Used to create a new ‘Referrer’ in Greenhouse if referrer.email does not match an existing user. Only required if referrer is included.
 notes | String | No | Free-form plain-text notes about this candidate. One way for this to be used is to send secondary information that our API can’t capture as structured data. For example: "Skills: Java, C++, Python"
+prospect_pool_id | Integer | Mixed | Used to put a prospect in the Prospect Pool with this id. Only required if prospect_pool_stage_id is included. prospect property must be set to true.
```

Comment:
Can we capitalize the start of the second sentence in these three properties?

Reply:
I didn't capitalize it bc the property name in the docs is also not capitalized. Maybe i'll say "property"